extension/llm/export/quantizer_lib.py

```diff
@@ -153,12 +153,139 @@ def get_qnn_quantizer(
             QnnQuantizer,
             QuantDtype,
         )
+        from torch.ao.quantization.observer import MinMaxObserver
 
     except ImportError:
         raise ImportError(
             "Please install the Qualcomm backend follwing https://pytorch.org/executorch/main/build-run-qualcomm.html"
         )
 
+    def annotate_matmul_16a8w(gm: torch.fx.GraphModule) -> None:
```

Comment:
Is it qnn specific or llama specific, should we move it to qualcomm custom quantization path somewhere? Just so it's easier to control the length of this file...

Reply:
Got it. Thanks for your advice.
I create the file in backends/qualcomm/quantizer/custom_annotation.py.